plaso/parsers/azure_applicationgatewayaccess.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
+"""Parser for Azure application gateway access logging saved to a file."""
+
+import json
+import json.decoder as json_decoder
```

Comment:
Since you are only using JSONDecodeError from this class, you could just do:
```from json.decoder import JSONDecodeError``` like some of the other JSON data parsers (e.g. https://github.com/log2timeline/plaso/blob/fe2e316b8c76a0141760c0f2f181d84acb83abc2/plaso/parsers/gcp_logging.py#L5)